## RCV1 - v1,v2 multilingual 
https://archive.ics.uci.edu/ml/datasets/Reuters+RCV1+RCV2+Multilingual,+Multiview+Text+Categorization+Test+collection#

In [56]:
import pickle
from main import RCV_Document

In [57]:
pickle_in = open("RCV1v2_fetch/rcv1_news_train.pickle","rb")
rcv1_news_train = pickle.load(pickle_in)

In [58]:
len(rcv1_news_train)

23149

In [59]:
pickle_in = open("RCV1v2_fetch/rcv1_news_test.pickle","rb")
rcv1_news_test = pickle.load(pickle_in)

In [60]:
len(rcv1_news_test)

781265

In [61]:
# pickle RICK!!!

In [66]:
print("title: ",rcv1_news_train[3].title)
print("headline: ",rcv1_news_train[3].headline)
# print(rcv1_news_train[3].content)

title:  USA: CompuServe reports loss, cutting work force.
headline:  CompuServe reports loss, cutting work force.


In [76]:
empty_title = 0
for d in rcv1_news_test:
    if d.headline == '':
        empty_title += 1

In [77]:
empty_title

8

In [78]:
print("title: ",rcv1_news_test[3].title)
print("headline: ",rcv1_news_test[3].headline)

title:  UK: CYCLING-ROUSSEAU COMPLETES UNIQUE DOUBLE.
headline:  CYCLING-ROUSSEAU COMPLETES UNIQUE DOUBLE.


[TRAIN] read 23149 documents in rcv1-v2, and 101 labels

[TEST] read 781265 documents in rcv1-v2, and 103 labels